.github/workflows/test.yml

```diff
@@ -9,30 +9,31 @@ jobs:
   test:
     name: Run NPM Test (e2e)
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [13.x]
-        mongodb-version: [4.2]
+    container: node:12-alpine
+    services:
+      postgres:
+        image: postgres:13-alpine
+        env:
+          POSTGRES_DB: db_test
+          POSTGRES_USER: admin
+          POSTGRES_PASSWORD: admin
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
curious why we don't need this now?

Reply:
Turns out that there’s a better way to do that without the need for an Action